style.css

```diff
@@ -89,3 +90,303 @@ header {
   list-style: none;
   width: 60%;
 }
+
+.projects {
+  margin: 30% 10px 10px 10px;
+}
+
+.recent-project-name {
+  font-family: "Crete Round", serif;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  text-align: left;
+  margin: 15px 0;
+}
+
+.recent-project-img {
+  width: 100%;
+  margin-top: 15%;
+}
+
+.recent-project {
+  margin: 0 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.name {
+  font-family: "Crete Round", serif;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  color: #fff;
+  text-align: left;
+}
+
+.recent-project-note {
+  font-family: "Inter", serif;
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  color: #091e42;
+  margin-bottom: 20px;
+}
+
+.recent-project-tech {
+  display: flex;
+  justify-content: space-evenly;
+  margin: 20px 0;
+  list-style: none;
+}
+
+.recent-project-tech-used {
+  border: 1px solid #babfc9;
+  padding: 5px 12px;
+}
+
+.see-recent-project {
+  background-color: #ff6b00;
+  color: #fff;
+  padding: 10px 20px;
+  border: none;
+}
+
+.projects-done {
+  margin: 30px 0;
+  background-image: url(./icons/Img-Placeholder.svg);
+  height: 100%;
+  background-repeat: no-repeat;
+  display: flex;
+  justify-content: flex-end;
+  flex-direction: column;
+  background-size: cover;
+  width: 100%;
+}
+
```

Comment:
- [ ] I would suggest you to add padding in the cards as the content seems to be collapsing into the border of the container. 

|  Figma Design  |  Your Design |
|----------------|--------------|
|<img width="359" alt="Screenshot 2023-04-30 at 2 24 39 AM" src="https://user-images.githubusercontent.com/44798044/235325072-20d66bb4-ad32-4176-a845-41dc154b16fa.png">|<img width="369" alt="Screenshot 2023-04-30 at 2 25 25 AM" src="https://user-images.githubusercontent.com/44798044/235325182-bdc6f70d-5221-4157-b888-52c9b27fcca5.png">|


- [ ] Also please add margin between the cards to match the layout of the[ design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=48-27&t=bpXoHDvfJoc09qP5-0).

|  Figma Design  |  Your Design |
|----------------|--------------|
<img width="263" alt="Screenshot 2023-04-30 at 2 25 02 AM" src="https://user-images.githubusercontent.com/44798044/235325124-31767b29-5c7a-441d-ab74-36aa4e9a53af.png">|<img width="369" alt="Screenshot 2023-04-30 at 2 25 25 AM" src="https://user-images.githubusercontent.com/44798044/235325237-850cb178-3865-47ab-aa10-1d438a24f284.png">|